CHANGELOG.rst

```diff
@@ -98,6 +98,10 @@ New features:
 - Allow usage of ``F`` expressions to in annotations. (#301)
 - Now negative number with ``limit(...)`` and ``offset(...)`` raise ``ParamsError``. (#306)
 
+* Add distinct to Aggregate
+
+  Functions that inherits Aggregate can use with ``distinct`` flag
+
```

Comment:
This CHANGELOG is also technically correct. but I think average user will not see the implication. 
Rather say `Add ability to supply ``distinct`` flag to Aggregate`? Just one line looks enough.

Reply:
Done.
Thx for your suggestion